src/common/util.js

```diff
@@ -144,13 +180,26 @@ function skipNewline(text, index, opts) {
   return index;
 }
 
+/**
+ * @param {string} text
+ * @param {number} index
+ * @param {SkipOptions=} opts
+ * @returns {boolean}
+ */
 function hasNewline(text, index, opts) {
   opts = opts || {};
+  // @ts-ignore => TBD: false - 1 is not allowed
```

Comment:
I think this @ts-ignore comment is not needed; `yarn check-types` seems OK with this comment removed.

Reply:
That’s correct! We forgot to remove it when this problem was fixed.